tests/multipleBackend/objectPut.js

```diff
@@ -132,8 +132,9 @@ describeSkipIfE2E('objectPutAPI with multiple backends', function testSuite() {
         });
     });
 
-    it('should put an object to file based on request endpoint', done => {
-        put(null, null, 'localhost', () => {
+    it('should put an object to us-east-1 based on bucket location if ' +
```

Comment:
why change? (should at least say us-east-1 which is file)

Reply:
This test was not testing `putting an object to file based on request endpoint` because when creating a bucket without any location => bucket location === us-east-1
So object's data will be stored in us-east-1 location if object location constraint is empty.
Note: I m gonna replace undefined to null, it actually does not matter.